neptune/new/run.py

```diff
@@ -230,5 +233,7 @@ def _define_attribute(self, _path: List[str], _type: AttributeType):
         if _type == AttributeType.RUN_STATE:
             self._structure.set(_path, RunStateAttr(self, _path))
 
+        raise NeptuneException(f"Unexpected type: {_type}")
```

Comment:
Not 100% sure that we can raise exception here, but for sure we shouldn't let this function go through silently without taking any action.

Reply:
Exception is fine here. But we should fix hosted_neptune_backend:get_attributes:320 to ignore unknown types so adding them in backend won't break old client.